powerhome/playbook
# Remove jQuery from docs site

#### What does this PR do?

1. Removes jQuery from the docs site and replaces jQuery code with vanilla js code.
2. Fixes small null handling bug in dark mode toggle (component breaks when dark mode cookie is undefined).

#### Breaking Changes

No, original jQuery behavior replaced with vanilla js.

#### Runway Ticket URL

[Link to Runway Story](https://nitro.powerhrg.com/runway/backlog_items/NUXE-203)

#### How to test this

Make sure code toggle still works in doc examples (only piece of jQuery that was replaced).

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
